configuration/configuration.go

```diff
@@ -811,6 +813,15 @@ func (c *Registry) GetTogglesServiceURL() string {
 	return c.v.GetString(varTogglesServiceURL)
 }
 
+// GetDeploymentsTimeout returns the amount of seconds until it should timeout.
+func (c *Registry) GetDeploymentsHttpTimeoutSeconds() time.Duration {
+	timeout := c.v.GetInt(varDeploymentsHttpTimeout)
+	if timeout < minimumDeploymentsHttpTimeout {
+		timeout = minimumDeploymentsHttpTimeout
+	}
+	return time.Duration(timeout) * time.Second
```

Comment:
method GetDeploymentsHttpTimeoutSeconds should be GetDeploymentsHTTPTimeoutSeconds